doc/nrf/releases_and_maturity/releases/release-notes-2.7.99-cs1.rst

```diff
@@ -0,0 +1,112 @@
+
+Added the following feature as experimental:
+
+* New peripheral driver SAADC.
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v2.7.99-cs1**.
+Check the :file:`west.yml` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`cloning_the_repositories` and :ref:`gs_updating_repos_examples` for more information.
+
+For information on the included repositories and revisions, see `Repositories and revisions for v2.7.99-cs1`_.
+
+IDE and tool support
+********************
+
+`nRF Connect extension for Visual Studio Code <nRF Connect for Visual Studio Code_>`_ is the recommended IDE for |NCS| v2.7.99-cs1.
+See the :ref:`installation` section for more information about supported operating systems and toolchain.
+
+.. note::
+
+   Because of the significant, potentially breaking changes introduced in the |NCS| v2.7.0, the support for v2.7.99-cs1 in the |nRFVSC| is experimental in the following areas:
+
+     * sysbuild - During the extension stabilization period, manually edit and verify Kconfig option values instead of using the nRF Kconfig GUI.
+     * Hardware Model v2 - During the extension stabilization period, edit devicetree files manually instead of using the Devicetree Visual Editor.
+       Moreover, custom boards added in the extension will follow the Zephyr's Hardware Model v1.
+
+   The extension users that need v2.7.99-cs1 should `switch to the pre-release version of the extension`_.
+
+Migration notes
+***************
+
+See the `Migration notes for nRF Connect SDK v2.7.99-cs1 and the nRF54H20 DK`_ for the changes required or recommended when migrating your environment to the |NCS| v2.7.99-cs1.
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+Zephyr
+======
+
+.. NOTE TO MAINTAINERS: All the Zephyr commits in the below git commands must be handled specially after each upmerge and each nRF Connect SDK release.
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``a415e2d702d5192ffdf7ddf67fa73f8fab70a277``, with some |NCS| specific additions.
+
+* Added:
+
+  * Power management - Support for System On All Idle power state.
+  * Experimental support for a new peripheral driver SAADC.
+    See the :zephyr:code-sample:`adc_sequence` and :zephyr:code-sample:`adc_dt` sample documentation for more details.
+  * Support for the clock control API.
+    See the :file:`include/zephyr/drivers/clock_control/nrf_clock_control.h` file for more details.
+
+For the list of upstream Zephyr commits (not including cherry-picked commits) incorporated into nRF Connect SDK since the most recent release, run the following command from the :file:`ncs/zephyr` repository (after running ``west update``):
+
+.. code-block:: none
+
+   git log --oneline a415e2d702 ^e7415e4e39
```

Comment:
@gmarull : Please check whether the commit "e7415e4e39" is correct.

Reply:
looks good